fe/fe-core/src/main/java/com/starrocks/transaction/LakeTableTxnStateListener.java

```diff
@@ -146,8 +145,8 @@ public void postAbort(TransactionState txnState) {
             request.tabletIds = entry.getValue();
 
             try {
-                LakeServiceAsync lakeService = BrpcProxy.getLakeService(backend.getHost(), backend.getBrpcPort());
-                lakeService.abortTxn(request, new EmptyRpcCallback<>());
+                LakeService lakeService = BrpcProxy.getLakeService(backend.getHost(), backend.getBrpcPort());
+                lakeService.abortTxn(request).get();
```

Comment:
```suggestion
                lakeService.abortTxn(request);
```
It's ok to  ignore the  response.